x-pack/plugins/code/public/components/main/side_tabs.tsx

```diff
@@ -26,9 +26,25 @@ enum Tabs {
 }
 
 const Container = styled.div`
+  width: calc(256rem / 14);
   border-right: ${euiBorderThin};
   display: flex;
   flex-direction: column;
+  & > div {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    & > div:first-child {
```

Comment:
https://github.com/elastic/kibana/pull/29326/commits/cc63d9bc967832ed6cbd57dfa350059d1b842717#diff-9c756f8e74dfbc1375a7e5b33139bef2R21

This is the one used in the search page. The simple idea is to use 50% as the width? Do we need to have these complex settings?